Added init file and corrected flake 8 errors

MikeS-1 opened the pull request.

ldconejo commented:
Hi Mike,

Unit and integration test for both calculator and water regulation are working ok. Pylint for water regulation shows a few issues:

************* Module waterregulation.sensor.sensor
waterregulation/sensor/sensor.py:16:0: C0303: Trailing whitespace (trailing-whitespace)
waterregulation/sensor/sensor.py:20:0: C0303: Trailing whitespace (trailing-whitespace)
waterregulation/sensor/sensor.py:27:0: C0303: Trailing whitespace (trailing-whitespace)
waterregulation/sensor/sensor.py:28:0: C0301: Line too long (109/100) (line-too-long)
waterregulation/sensor/sensor.py:8:0: R0903: Too few public methods (1/2) (too-few-public-methods)
************* Module waterregulation.sensor.__init__
waterregulation/sensor/__init__.py:1:0: C0304: Final newline missing (missing-final-newline)
waterregulation/sensor/__init__.py:1:0: C0111: Missing module docstring (missing-docstring)
************* Module waterregulation.pump.__init__
waterregulation/pump/__init__.py:1:0: C0304: Final newline missing (missing-final-newline)
************* Module waterregulation.pump
waterregulation/pump/__init__.py:1:0: C0111: Missing module docstring (missing-docstring)

Please take a look at those and try to fix as many as possible. If you are using a custom .pylintrc file make sure it's included in the commit.

Thanks,

Luis

MikeS-1 commented:
https://github.com/UWPCE-PythonCert-ClassRepos/SP_Online_Course2_2018/pull/170


Luis,

Thanks for the feedback. I think I’m running into a version control issue with git. I don’t get the same errors you’re showing, and I think it’s because I moved the sensor and pump file to the main folder and wrote a single init file for the package. I haven’t been able to delete the folders, though. Also, for whatever reason, the .pylintrc file doesn’t upload on git. I’ve attached it to this email. Please let me know what I can do better, and thanks again.

V/r

Mike

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________
From: Luis Conejo-Alpizar <notifications@github.com>
Sent: Monday, August 13, 2018 10:55:49 PM
To: UWPCE-PythonCert-ClassRepos/SP_Online_Course2_2018
Cc: MikeS-1; Author
Subject: Re: [UWPCE-PythonCert-ClassRepos/SP_Online_Course2_2018] Added init file and corrected flake 8 errors (#165)


Hi Mike,

Unit and integration test for both calculator and water regulation are working ok. Pylint for water regulation shows a few issues:

************* Module waterregulation.sensor.sensor
waterregulation/sensor/sensor.py:16:0: C0303: Trailing whitespace (trailing-whitespace)
waterregulation/sensor/sensor.py:20:0: C0303: Trailing whitespace (trailing-whitespace)
waterregulation/sensor/sensor.py:27:0: C0303: Trailing whitespace (trailing-whitespace)
waterregulation/sensor/sensor.py:28:0: C0301: Line too long (109/100) (line-too-long)
waterregulation/sensor/sensor.py:8:0: R0903: Too few public methods (1/2) (too-few-public-methods)
************* Module waterregulation.sensor.init
waterregulation/sensor/init.py:1:0: C0304: Final newline missing (missing-final-newline)
waterregulation/sensor/init.py:1:0: C0111: Missing module docstring (missing-docstring)
************* Module waterregulation.pump.init
waterregulation/pump/init.py:1:0: C0304: Final newline missing (missing-final-newline)
************* Module waterregulation.pump
waterregulation/pump/init.py:1:0: C0111: Missing module docstring (missing-docstring)

Please take a look at those and try to fix as many as possible. If you are using a custom .pylintrc file make sure it's included in the commit.

Thanks,

Luis

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUWPCE-PythonCert-ClassRepos%2FSP_Online_Course2_2018%2Fpull%2F165%23issuecomment-412737796&data=02%7C01%7C%7C900b65d79c624f6c901a08d60191715a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636698121507975056&sdata=9oNm0Jz1hohibXiJW4R2ckLWx1DJhurjSXae%2FeiSORk%3D&reserved=0>, or mute the thread<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAid7TDus-wN9f3RkL_mHqL4GbcSVGyhfks5uQjw1gaJpZM4V5z2x&data=02%7C01%7C%7C900b65d79c624f6c901a08d60191715a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636698121508131316&sdata=WxacdYh4XUHLMpgCmOnZnxmRuSpQzU1SngDLZ4RHrWY%3D&reserved=0>.
